DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
 `Last update: 2018-07`
 
-# Service owners
+# _Mozilla IAM_ Program Strategy
 
-High level service and product owners for Mozilla IAM.
+* @jeffbryner (IT, InfoSec)
+* @hmitsch (Open Innovation, ParSys)
 
-* Capability owner: @jeffbryner
-  * Product owner: @gdestuynder
+# Mozilla IT Operating Model
+
+* IT Capability Owner: @jeffbryner
+  * IT Product Owner: @gdestuynder
     * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+       * <Question: Is `Auth0 Lock` the same as `NLX`>
     * Service Integration (CIS): @akrug
     * LDAP: @jdow
     * DuoSecurity: @gdestuynder
-    * Mozillians.org: @hmitsch
-    * WorkDay: HRIS
-
-## Architects
-
-* @gdestuynder (IAM, Security)
-* @mvk-mozilla (IT)
-
-## Program Managers
-
-* @hmitsch
-
-## User Experience (UX)
-
-* @mbranson
+    
+* IT Enterprise Architecture
+   * @mvk-mozilla
+   
+* Outside-IT Capability Ownership
+    * User Experience (UX): @mbranson
+    * Mozillians.org: ParSys
+    * WorkDay (HRIS): People Team
 
-# Modules owners
+# Modules Owners
 
-This document list which team or persons are responsible for which component (module) of the Mozilla IAM stack. This is
-a more detailed model.
+This document list which team or people are responsible for which component (module) of the _IAM Stack_. This is mainly useful for decision making and is inspired by [Mozilla's Module System](https://wiki.mozilla.org/Modules).
 
-This is mainly useful for decision making and is loosely following the [Firefox Module
-system](https://wiki.mozilla.org/Modules).
+_Note:_ We don't yet know what a conflict resolution process would look like if there was a diverging opinion between the Module Owner and the IT Capability Owner for that module.
```

Comment:
im guessing its a note/question, not something to add to the doc
I dont think the 2 diverge - for NLX, if there is a problem it comes up to the service, then product, then capability owner to decide.

Reply:
I think this is where we hit the barriers of the Mozilla Module System and the Mozilla IT Service Ownership Model. I don't think that these two systems mix very well.

Reply:
see jeff comments below, i think he has a good solution